sklearn/ensemble/gradient_boosting.py

```diff
@@ -1406,7 +1417,8 @@ class GradientBoostingClassifier(BaseGradientBoosting, ClassifierMixin):
     def __init__(self, loss='deviance', learning_rate=0.1, n_estimators=100,
                  subsample=1.0, criterion='friedman_mse', min_samples_split=2,
                  min_samples_leaf=1, min_weight_fraction_leaf=0.,
-                 max_depth=3, min_impurity_split=1e-7, init=None,
+                 max_depth=3, min_impurity_decrease=0.,
```

Comment:
Thanks for the catch. I changed the doc to 0... I'm using 0 because of the [`EPSILON` added as described here](https://github.com/scikit-learn/scikit-learn/pull/8449#discussion_r103059448)...